torch/csrc/jit/import.cpp

```diff
@@ -305,7 +305,7 @@ at::Tensor ModuleDecoder::buildTensorCommon(
     int64_t size;
     std::tie(storage_ptr, size) = file_reader_.getRecordWithKey(record_number);
     auto storage = std::make_shared<at::Storage>(
-      at::CPU(type).scalarType(),
+      at::CPU(type).typeMeta(),
```

Comment:
This use site really needs to get rewritten, so that it's not running `make_shared` on Storage (which itself is a shared pointer). You don't have to fix it here, but something to note.